depends/config.site.in

```diff
@@ -1,4 +1,14 @@
-depends_prefix="`dirname ${ac_site_file}`/.."
+# shellcheck shell=sh disable=SC2034 # Many variables set will be used in
```

Comment:
20359 doesn't seem partial

Reply:
It's not, was marked partial when debugging `PYTHONPATH` errors, then reverted without reverting back message, resolved